Danki2/Assets/Scripts/Abilities/InstantCast/Slash.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Abilities;
+
+namespace Assets.Scripts.Abilities.InstantCast
+{
+    public class Slash : InstantCast
+    {
+        public Slash(AbilityContext context) : base(context)
```

Comment:
Just wondering if you'd want to be able to add configuration to the ability through the constructor, or do you think the ability itself will just go and look things up from the stats on the Actor in the context? If so then it's all fine

Reply:
I'm thinking we can just look stuff up on the Actor yeah.